types/compression-webpack-plugin/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for compression-webpack-plugin 2.0
+// Type definitions for compression-webpack-plugin 2.1
```

Comment:
From what I can tell, I don’t think a version bump is needed. It sounds like this is passed through to Node, which means it would have worked in 2.0 as well as 2.1+.